mainwindow.cpp

```diff
@@ -94,6 +99,14 @@ namespace dealii
 
 
 
+    void MainWindow::set_to_default()
+    {
+      QTreeWidgetItem * current_item = tree_widget->currentItem();
+      current_item->setText(1,current_item->text(2));
```

Comment:
Here I just retrieve a pointer to an existing item that belongs to the tree_widget. But in general every Widget can have a parent (With the outermost parent being the main window). I suppose, but have not checked, that every parent is responsible for deleting its children. That being said, I am not sure all of our widgets specify a parent, and dont know what happens if they use the default (*parent=0). Do you want to open an issue for this?